contrib/azure/create-azure-user-data

```diff
@@ -56,14 +56,19 @@ def main():
     azure_template = get_file("azure-user-data-template")
     coreos_template = get_file("../coreos/user-data.example")
 
-    configuration_coreos_template = yaml.safe_load(coreos_template)
+    coreos_template_w_url = None
+    with open(coreos_template.name, 'r') as file :
+      coreos_template_w_url = file.read()
+
+    coreos_template_w_url = coreos_template_w_url.replace('#DISCOVERY_URL', url)
```

Comment:
With newer versions of PyYAML the "safe_load" would set the `discovery` variable in cloud-config to "none" and remove the comment, making this logic ineffective. This was my workaround.